docs/install/installUsingDocker.md

```diff
@@ -1,67 +1,47 @@
-
-### Supported operators
-
-* Percona Operator for MySQL Based on Percona XtraDB Cluster (PXC) 1.13.0
-* Percona Operator for MongoDB (PSMDB) 1.14.0
-* Percona Operator for PostgreSQL (PG) 2.2.0
-
-### Supported k8s clusters
-
-* Google Kubernetes Engine (GKE) 
-* Amazon Elastic Kubernetes Service (EKS) (1.24 - 1.25)
-
-## Before you start
-
-Percona Everest assists with installing all the necessary operators and required packages, but does not currently help with spinning up a publicly accessible Kubernetes cluster.
-
-We recommend setting up Percona Everest on the [Amazon Elastic Kubernetes Service (EKS)](../quickstart-guide/eks.md) or on [Google Kubernetes Engine (GKE)](../quickstart-guide/gke.md), as Percona Everest may not work as expected on local Kubernetes installations (minikube, kind, k3d, or similar products) due to network issues.
-
-## Prerequisites
-
-Before getting started with Percona Everest, we recommend that you:
+To install and provision Percona Everest using Docker Compose:
 {.power-number}
 
-1. Install [Docker Engine](https://docs.docker.com/engine/install){:target="_blank"} (1.13.0 and higher) with the [Docker compose plugin](https://docs.docker.com/compose/install/){:target="_blank"}.
-2. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html){:target="_blank"} for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install){:target="_blank"} for GKE.
-3. Verify that you have access to the Kubernetes cluster that you want to use with Everest. By default, Everest uses the kubeconfig file available under *~/.kube/config*. Run the following command:
-    
-    ```sh 
-    kubectl get nodes
+1. Download the Docker Compose file:
+   
     ```
+    curl https://raw.githubusercontent.com/percona/percona-everest-backend/release-0.4.0/deploy/quickstart-compose.yml -o quickstart-compose.yml
+    ```
+
+2. Create the `percona-everest` namespace:
 
+    ```sh
+    kubectl create namespace percona-everest
+    ```
+3. Create a kubernetes secret with an auto-generated root key used for encrypting secrets:
+   
+    ```sh
+    ENCODED_SECRETS_ROOT_KEY=$(openssl rand -base64 32 | tr -d '\n' | base64); cat <<EOF | envsubst | kubectl apply -n percona-everest -f -
+    apiVersion: v1
+    kind: Secret
+    metadata:
+        name: everest-secrets-root-key
+    data:
+        secrets-root-key: $ENCODED_SECRETS_ROOT_KEY
+    EOF
+    ```
+    
     ??? example "Expected output"
-        ```
-            NAME                                       STATUS   ROLES    AGE   VERSION
-            gke-<name>-default-pool-75d48bfc-bx8g      Ready    <none>   11h   v1.26.7-gke.500
-            gke-<name>-default-pool-75d48bfc-c2df      Ready    <none>   11h   v1.26.7-gke.500
-            gke-<name>-default-pool-75d48bfc-zl7k      Ready    <none>   11h   v1.26.7-gke.500
+    
+        ```{.text .no-copy}
+        secret/everest-secrets-root-key configured
         ```
```

Comment:
@rasika-chivate I think you misunderstood me here. These were not the two commands I was talking about.
This is what we want to add here and in the quickstart guide.
https://github.com/percona/percona-everest-backend/blob/dd03865980cf7abda1b5d705476a5be0bdf21464/README.md?plain=1#L23-L29

Reply:
Fixed